src/CMakeLists.txt

```diff
@@ -54,13 +54,17 @@ include(GetTargetInterface)
 # -fsanitize and related flags apply to both C++ and C,
 # so we can pass them down to libsecp256k1 as CFLAGS and LDFLAGS.
 get_target_interface(core_sanitizer_cxx_flags "" sanitize_interface COMPILE_OPTIONS)
-set(SECP256K1_APPEND_CFLAGS ${core_sanitizer_cxx_flags} CACHE STRING "" FORCE)
```

Comment:
In 21bfce58c1cad31801e045c606bd8fb16a00e11d: Just generally, I'd say it's a bit less ideal, having secp256k1 flags/config getting split up into multiple (arbitrary) places. Especially if in future, we'll have to duplicate all this logic in all places, for all subtrees (when using their own cmake build systems).